scripts/build.py

```diff
@@ -598,17 +601,110 @@ def build_files(source, target, file_hashes):
                 continue
             target_path = source_path.replace(source, target)
             ensure_directory_exists(target_path)
-
-            task = threading.Thread(
-                target=minify_func,
-                args=(source_path, target_path, file_hashes, filename))
+            if file_formats:
+                if any(filename.endswith(p) for p in file_formats):
+                    task = threading.Thread(
+                        target=minify_func,
+                        args=(source_path, target_path, file_hashes, filename))
+                else:
+                    # Skip files that are not the specified format.
+                    continue
+            elif file_formats is None:
```

Comment:
This could just be "else", right?

Reply:
yep. Reconfigured awkward if-else logic.